src/rules/utils/selectors/index.ts

```diff
@@ -23,3 +23,6 @@ export const ngModuleDecorator = `ClassDeclaration > Decorator > CallExpression[
 export const ngModuleProviders = `${ngModuleDecorator} ObjectExpression Property[key.name='providers'] > ArrayExpression Identifier`
 
 export const ngModuleImports = `${ngModuleDecorator} ObjectExpression Property[key.name='imports'] > ArrayExpression CallExpression[callee.object.name='EffectsModule'][callee.property.name=/forRoot|forFeature/] ArrayExpression > Identifier`
+
+export const selectWithLiteral = `CallExpression[callee.property.name="pipe"] CallExpression[callee.name="select"] Literal`
```

Comment:
After a second look, we can create another query that we could reuse as a `select` query select 

```
CallExpression[callee.property.name="pipe"] CallExpression[callee.name="select"]
```

While I was think about this, I also thought that we should support `select` function on the `store`, e.g. `store.select(selectCustomers)`. Do you also want to add it to this PR?

Reply:
Good idea!
I added support for store select and refactored a little to share eslint selectors

Reply:
I can even make one selector for everything actually